File: programme/simple-interest-calculator/simple-interest-calculator.c
// Write a programme to calculate simple interest
#include <stdio.h>

int simple_interest(int p, int t, int r)
{
    return (int)(p * t * r / 100);
}

int main(void)
{
    int p, t, r;
    printf("Enter P : ");
    scanf("%d", &p);
    printf("Enter R : ");
    scanf("%d", &r);
    printf("Enter T : ");
    scanf("%d", &t);
    printf("\nSimple Interest : %d\n", simple_interest(p, t, r));
    return 0;
}

File: programme/simple-interest-calculator/simple-interest-calculator.js
// Write a programme to calculate simple interest

"use strict";

// perform following command to run programme
// node simple-interest-calculator.js

const readline = require("readline");

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout,
});

rl.question("Enter P : ", (p) => {
  rl.question("Enter T : ", (t) => {
    rl.question("Enter R : ", (r) => {
      const si = (parseInt(p) * parseInt(t) * parseInt(r)) / 100;
      if (isNaN(si)) {
        console.error("Some error occured");
      } else {
        console.log(`\nSimple Interest : ${si}`);
      }
      rl.close();
    });
  });
});
